jsonnet/telemeter/server/kubernetes.libsonnet

```diff
@@ -135,6 +135,7 @@ local clusterPort = 8082;
       local secret = k.core.v1.secret;
 
       secret.new(secretName, {
+        authorize_url: std.base64($._config.telemeterServer.authorizeUrl),
```

Comment:
With these changes Jsonnet will fail build this because there is no field named `authorize_url` in the config. How about leaving it as `''`?

Reply:
fixed